src/video/SDL_egl.c

```diff
@@ -539,7 +558,7 @@ SDL_EGL_LoadLibrary(_THIS, const char *egl_path, NativeDisplayType native_displa
         return SDL_SetError("Could not get EGL display");
     }
 
-    if (_this->egl_data->eglInitialize(_this->egl_data->egl_display, NULL, NULL) != EGL_TRUE) {
+    if (_this->egl_data->eglInitialize(_this->egl_data->egl_display, &major, &minor) != EGL_TRUE) {
```

Comment:
It looks like these variables are never read

Reply:
It was a leftover from my debugging session. Should be removed.

Reply:
I mean, restored with original code.